app/models/agents/gap_detector_agent.rb

```diff
@@ -8,6 +8,11 @@ class GapDetectorAgent < Agent
       The `value_path` value is a [JSONPath](http://goessner.net/articles/JsonPath/) to a value of interest. If either
       this value is empty, or no Events are received, during `window_duration_in_days`, an Event will be created with
       a payload of `message`.
+
+      The `stream_path` value is a [JSONPath](http://goessner.net/articles/JsonPath/) to a value in the event that will
```

Comment:
Just to play devil's advocate.  What if you wanted to utilize both?  For instance, detect gaps based on a "hostname", but only events which are "successful."